src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,170 @@
+@include govuk-exports("govuk/component/pagination") {
+  .govuk-pagination {
+    border-top: 1px solid $govuk-border-colour;
+    text-align: center;
+
+    @include govuk-media-query($from: tablet) {
+
+      // Hide whitespace between elements
+      font-size: 0;
+
+      // Trick to remove the need for floats
+      text-align: justify;
+
+      &:after {
+        content: '';
+        display: inline-block;
+        width: 100%;
+      }
+    }
+  }
+
+  .govuk-pagination__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    @include govuk-media-query($from: tablet) {
+      display: inline-block;
+      margin-bottom: 0;
+    }
+  }
+
+  .govuk-pagination__results {
+    @include govuk-font(19);
+    margin-top: 0;
+    margin-bottom: govuk-spacing(4);
+    padding: govuk-spacing(1) 0;
+    @include govuk-media-query($from: tablet) {
+      display: inline-block;
+      margin-bottom: 0;
+      //vertical-align: middle;
+    }
+  }
+
+  .govuk-pagination__item {
+    @include govuk-font(19);
+    display: inline-block;
+    margin-bottom: govuk-spacing(4);
+
+    // Hide items on small screens
+    @include govuk-media-query($until: tablet) {
+      display: none;
+    }
+
+    .govuk-pagination__icon {
+      margin-top: 6px;
+      margin-right: govuk-spacing(3);
+      fill: $govuk-secondary-text-colour;
+    }
+  }
+
+  .govuk-pagination__item--next .govuk-pagination__icon {
+    margin-right: 0;
+    margin-left: govuk-spacing(3);
+    float: right;
+  }
+
+  // Only show previous, next, first, last and current items on mobile
+  .govuk-pagination__item--active,
+  .govuk-pagination__item--elipses,
+  .govuk-pagination__item--prev,
+  .govuk-pagination__item--next,
+  .govuk-pagination__item:nth-child(2),
+  .govuk-pagination__item:nth-last-child(2) {
+    @include govuk-media-query($until: tablet) {
+      display: inline-block;
+    }
+  }
+
+  .govuk-pagination__item--active,
+  .govuk-pagination__item--elipses {
+    box-sizing: border-box;
+    min-width: 45px;
+    min-height: 45px;
+    padding: 10px 18px;
+    color: $govuk-secondary-text-colour;
+    font-weight: bold;
+    text-align: center;
+  }
+
+  .govuk-pagination__item--elipses {
+    margin: 0 -22.5px;
+    padding-right: 0;
+    padding-left: 0;
+  }
```

Comment:
Should the current page number appear black rather than dark grey?

<img width="995" alt="Screenshot 2021-11-18 at 23 21 05" src="https://user-images.githubusercontent.com/30665/142512252-2cf19ac0-088a-428a-87ec-1559a81561bd.png">

vs

<img width="989" alt="Screenshot 2021-11-18 at 23 14 35" src="https://user-images.githubusercontent.com/30665/142512065-35afa41b-46dd-423c-ba51-14439b0a6473.png">


```suggestion
    font-weight: bold;
    text-align: center;
  }

  .govuk-pagination__item--elipses {
    margin: 0 -22.5px;
    padding-right: 0;
    padding-left: 0;
    color: $govuk-secondary-text-colour;
  }
```